source/sqlsrv/stmt.cpp

```diff
@@ -1084,8 +1084,8 @@ PHP_FUNCTION( sqlsrv_get_field )
             THROW_SS_ERROR( stmt, SS_SQLSRV_ERROR_INVALID_FUNCTION_PARAMETER, _FN_ );
         }
 
-		core_sqlsrv_get_field( stmt, field_index, sqlsrv_php_type, false, field_value, &field_len, false/*cache_field*/,
-			&sqlsrv_php_type_out TSRMLS_CC );
+        core_sqlsrv_get_field( stmt, static_cast<SQLUSMALLINT>( field_index ), sqlsrv_php_type, false, field_value, &field_len, false/*cache_field*/,
+            &sqlsrv_php_type_out TSRMLS_CC );
```

Comment:
indentation, `&sqlsrv_php_type_out` should be under `stmt`, use spaces instead of tabs